src/components/Box.js

```diff
@@ -1,9 +1,10 @@
 import React, { PropTypes } from 'react'
 import EmailPropTypes from '../PropTypes'
+import includeDataProps from '../includeDataProp'
 
 export default function Box({ children, ...props }) {
   return (
-    <table {...props}>
+    <table {...includeDataProps(props)} {...props}>
```

Comment:
Is `includeDataProps` redundant here, since we're already passing through all `props`?